plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -417,10 +442,23 @@ def _set_event_string(event, s):
         kCGEventFlagMaskAlternate |
         kCGEventFlagMaskControl |
         kCGEventFlagMaskShift |
-        kCGEventFlagMaskCommand
+        kCGEventFlagMaskCommand |
+        kCGEventFlagMaskSecondaryFn
     )
 
     @staticmethod
+    def _send_media_key(key_id, key_down):
+        # Credit: https://gist.github.com/fredrikw/4078034
+        event = NSEvent.otherEventWithType_location_modifierFlags_timestamp_windowNumber_context_subtype_data1_data2_(
+            NSSystemDefined, (0,0),
+            0xa00 if key_down else 0xb00,
+            0, 0, 0, 8,
+            (key_id << 16) | ((0xa if key_down else 0xb) << 8),
```

Comment:
This could be simplified/factorized a little:
```python
	flags = 0xa00 if key_down else 0xb00
	event = NSEvent.otherEventWithType_location_modifierFlags_timestamp_windowNumber_context_subtype_data1_data2_(
	    NSSystemDefined, (0,0),
	    flags,
	    0, 0, 0, 8,
	    (key_id << 16) | flags,
	    -1
	)
	CGEventPost(0, event.CGEvent())
```